web/src/containers/activity/ContractorResources.js

```diff
@@ -52,7 +52,7 @@ const ContractorResources = ({
 };
 
 ContractorResources.propTypes = {
-  activityIndex: PropTypes.number.isRequired,
```

Comment:
wouldn't it be better to require this to be a number?  It will always be an integer or if it's a string (not sure how that happens) it can be converted

Reply:
`activityIndex` is extracted from the URL, which is a string.

Reply:
No, I'm wrong. Going up a few component levels, `<EntryPage />` coerces the activityIndex url param to a number. Will fix.

https://github.com/18F/cms-hitech-apd/blob/master/web/src/containers/activity/EntryPage.js#L9